File: src/test/scala/ro/dg/MorseCodeTest.scala
package ro.dg

import com.example.alg.stringpck.MorseCode
import org.junit.Assert.assertEquals
import org.junit.Test

class MorseCodeTest {

  @Test def uniqueMorseRepresentations(): Unit = {
    val morseCode = new MorseCode
    val uniqueMorseRepresentations =
      morseCode.uniqueMorseRepresentations(Array[String]("gin", "zen", "gig", "msg"))
    assertEquals(2, uniqueMorseRepresentations)
  }

}

File: src/test/java/com/example/alg/stringpck/MorseCodeTest.java
package com.example.alg.stringpck;

import org.junit.Test;

import static org.junit.Assert.*;

public class MorseCodeTest {

    @Test
    public void uniqueMorseRepresentations() {
        final MorseCode morseCode = new MorseCode();
        final int uniqueMorseRepresentations = morseCode.uniqueMorseRepresentations(new String[]{"gin", "zen", "gig", "msg"});
        assertEquals(2, uniqueMorseRepresentations);

    }
}